### PR TITLE
test cleanup

### PR DESCRIPTION
**Pull request by nec4**
Development:
 - [x] Implement feature / fix bug
 - [ ] Add documentation
 - [ ] Add tests
 
 Checks:
 - [x] Run `nosetests`
 - [x] Check pep8 compliance

Hello all. This PR is in response to #198 and addresses some intermittent test failures. In many places, I have raised the precisions of variables and adjusted certain tolerances. Two major fixes stand out:

1. For GPU tests, radial basis functions are now properly instanced so that the tests run properly.
2. In `test_simulation.test_saving_numpy_coordinates()`, the temporary filelist is now sorted (previously it was unsorted and this could lead to the loaded array being backwards). This does not affect the ability of simulations to be saved or loaded properly - it was just an issue with the test itself (and I think it might depend on how the `os.listdir()` sorts things by default).

All tests now pass for me and fave been PEP8 formatted. No changes to the main code have been made.

### CODE REVIEW

**Comment by nec4** [comment]
Once this is merged, I will address #199 and #197.

**Comment by nec4** [comment]
Yup! It's ready for review.

File: cgnet/tests/test_molecule_dataset.py [hunk 1]
```diff
@@ -157,48 +160,48 @@ def test_multi_molecule_collate():
...
+    # assemble the padded data into complete tensors of shape
+    # [frames, max_beads, 3] for coords/forces, and [frames, max_beads]
+    # for embeddings
+    manual_coords = torch.tensor(padded_coord_list, requires_grad=True)
+    manual_forces = torch.tensor(padded_force_list)
+    manual_embeddings = torch.tensor(padded_embedding_list)
+
+    # get tensors output from multi_molecule_collate()
+    coords, forces, embeddings = multi_molecule_collate(data)
+    print(coords.size())
```
Comment (brookehus) [inline cgnet/tests/test_molecule_dataset.py:200]: should this be here?

File: cgnet/tests/test_simulation.py [hunk 2]
```diff
@@ -509,19 +511,20 @@ def test_saving_numpy_coordinates():
...
-    n_expected_files = sim_length / npy_interval
-
-    model = HarmonicPotential(k=1, T=300, n_particles=10,
+    n_sims = 1  # np.random.randint(1, high=5)
+    sim_length = 4  # np.random.choice([24, 36])
+    npy_interval = 1  # np.random.choice([6, 12])
+    save_interval = 1  # np.random.choice([2, 3])
+
+    n_expected_files = int(sim_length / npy_interval)
+    print(n_expected_files)
```
Comment (brookehus) [inline cgnet/tests/test_simulation.py:520]: should this be here?

File: cgnet/tests/test_simulation.py [hunk 3]
```diff
@@ -533,8 +536,9 @@ def test_saving_numpy_coordinates():
 
         traj = my_sim.simulate()
         assert traj.shape[1] == sim_length / save_interval
-        file_list = os.listdir(tmp)
-
+        file_list = sorted(os.listdir(tmp))
+        print(len(file_list))
+        print(traj)
```
Comment (brookehus) [inline cgnet/tests/test_simulation.py:541]: should these print statements be here?

File: cgnet/tests/test_simulation.py [hunk 4]
```diff
@@ -589,7 +592,8 @@ def test_saving_all_quantities():
         assert traj.shape[1] == sim_length / save_interval
         file_list = os.listdir(tmp)
 
-        assert len(file_list) == n_expected_files * 4  # coords, forces, pot, ke
+        assert len(file_list) == n_expected_files * \
+            4  # coords, forces, pot, ke
```
Comment (brookehus) [inline cgnet/tests/test_simulation.py:596]: i would keep this the old way for readability

**Review by brookehus — CHANGES_REQUESTED** [review]
all looks good, thanks for doing that! just curious about a few print statements and one aesthetic thing

File: cgnet/tests/test_molecule_dataset.py (hunk 1 above)
Comment (nec4) [reply cgnet/tests/test_molecule_dataset.py]: Nope - removed!

File: cgnet/tests/test_simulation.py (hunk 2 above)
Comment (nec4) [reply cgnet/tests/test_simulation.py]: Nope - removed!

File: cgnet/tests/test_simulation.py (hunk 3 above)
Comment (nec4) [reply cgnet/tests/test_simulation.py]: Nope - removed!

File: cgnet/tests/test_simulation.py (hunk 4 above)
Comment (nec4) [reply cgnet/tests/test_simulation.py]: Good catch! Agreed.

**Comment by nec4** [comment]
Thanks for taking a look! I removed those print statements and reverted the assert line back to its original state. I reran the tests and they all pass for me. Let me know if there is anything else that can be changed or improved.

**Review by brookehus — APPROVED** [review]
awesome thank you!

**Pull request merged**